knack/iframeMapMessenger.js

```diff
@@ -0,0 +1,106 @@
+(function() {
+  console.log("iframeMapMessenger script begins...");
+  console.log(window.viewIdsArray);
+  var myView = window.viewIdsArray.shift(0);
+  console.log(myView);
```

Comment:
This was on me so I can come back in here and clean these. I don't think we need these console.logs anymore...

Reply:
No worries @mateoclarke - I just removed them and pushed. Merging away!